### PR TITLE
Added naming conventions for Python

### PR DESCRIPTION
**Pull request by PandaWhoCodes**
Edited /python/README.md
Added TL;DR table and a link to official Python naming conventions page.

Edited /python/variable-naming.md, /python/method-naming.md.
Added naming conventions for private/ non public methods/variables 

New Files for: 
Package Naming, Exception Naming, Uses of underscore in Python.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
   * [Contribute Guide](docs/CONTRIBUTING.md)
   * [FAQ](docs/faq.md)
   * [Lexicon](docs/lexicon.md)
-* [C#](csharp/README.md)
+* [C\#](csharp/README.md)
```
Comment (farbodsaraf) [inline SUMMARY.md]: @PandaWhoCodes May I ask why slash before #? Is it an escape character?

File: SUMMARY.md [hunk 2]
```diff
@@ -44,4 +44,8 @@
   * [Module Naming](python/module-naming.md)
   * [Method Naming](python/method-naming.md)
   * [Variable Naming](python/variable-naming.md)
-  * [Constant Naming](python/constant-naming.md)
\ No newline at end of file
+  * [Constant Naming](python/constant-naming.md)
+  * [Package Naming](python/file-naming.md)
+  * [Exception\( Error \) Naming](python/exception-naming.md)
```
Comment (farbodsaraf) [inline SUMMARY.md]: Same here, I see some escape characters. Is it necessary for the contents?

File: python/class-naming.md [hunk 3]
```diff
@@ -1,11 +1,27 @@
-# Python Naming Convention > Class Naming
+# Python Naming Convention &gt; Class Naming
```
Comment (farbodsaraf) [inline python/class-naming.md]: @PandaWhoCodes Is it necessary to turn all these signs to their HTTP code? e.g. > to &gt;
Would like to the reason cause if it's necessary I need to adjust other sections too. Thanks.

**Review by farbodsaraf — COMMENTED** [review]
Hi @PandaWhoCodes, can you please provide your thoughts on my comments below? Many thanks again for submitting this comprehensive PR.

File: SUMMARY.md (hunk 1 above)
Comment (PandaWhoCodes) [reply SUMMARY.md]: Hey,
All these changes are not intentional.
They were automatically done by my gitbook editor (desktop - windows).
The same with all the other changes you have pointed out.

Comment (farbodsaraf) [reply SUMMARY.md]: @PandaWhoCodes thanks for letting me know. By any chance, would it possible for you to revert them back? or should we merge your PR and start editing them back from the code?

**Pull request closed without merging**